tasks/winbuildscripts/lint.ps1

```diff
@@ -20,11 +20,23 @@ if($err -ne 0){
 }
 
 & inv -e install-tools
-& inv -e linter.go
 
+& inv -e linter.go
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Change can be undone